### ISSUE TITLE
shapely module should be a dependency - error thrown if not available

### ISSUE THREAD

**Issue by MattWenham** [maartenbreddels/ipyvolume#217]
When using `selector_default` as follows, if `shapely` is not available in the environment, an error is thrown. Installing the module eliminates the error.

    import ipyvolume as ipv
    import ipyvolume.pylab as p3

    fig = p3.figure(controls_vr=True)
    test = p3.scatter(
        embedding_unit_3d[:,0],
        embedding_unit_3d[:,1],    
        embedding_unit_3d[:,2],
        color = c,
        marker = 'sphere',
        size = 0.5
    )
    ipv.selector_default()
    p3.show()

**Comment by maartenbreddels** [comment]
yes I understand, but I don't want it to be a hard dependency, maybe it would have been nice to include it when doing
`pip install ipyvolume[all]`.
In the future I want the selection to be done in the browser, so shapely would not be required anymore.
